### ISSUE TITLE
Consider changing the name

### ISSUE THREAD

**Issue by ghostsquad** [maruina/argocd-progressive-rollout-controller#4]
This sounds almost exactly like how Argo Rollouts is described.

https://argoproj.github.io/argo-rollouts/

> Argo Rollouts - Kubernetes Progressive Delivery Controller

**Comment by maruina** [comment]
You are correct, do you have any suggestions? `argo rollouts` already has all the good buzzwords :)

**Comment by ghostsquad** [comment]
`argocd-promotion-controller` maybe?

**Comment by ghostsquad** [comment]
or maybe https://github.com/conveyr/delta ? I'd be happy to invite you. This was an idea I had before finding out about this controller.

**Comment by maruina** [comment]
I think we'll try to get some feedback from the community as soon as we get our of the POC state. We will propose `argocd-progressive-rollout` because I like being explicit with the concept of the progressive rollout.
